chartpress.py

```diff
@@ -232,6 +234,13 @@ def build_images(prefix, images, tag=None, commit_range=None, push=False, chart_
             'tag': SingleQuotedScalarString(image_tag),
         }
 
+        if skip_build:
+            continue
+
+        if tag is None and commit_range and not path_touched(*paths, commit_range=commit_range):
```

Comment:
This conditional should be removed, I think. This logic is below. Perhaps added by a sync with master?

Reply:
ah yes, most likely a rebase error, thanks